Orange/widgets/model/owneuralnetwork.py

```diff
@@ -135,6 +136,14 @@ def add_main_layout(self):
                 None, self, "max_iterations", 10, 10000, step=10,
                 label="Max iterations:", orientation=Qt.Horizontal,
                 alignment=Qt.AlignRight, callback=self.settings_changed))
+        
+        form.addRow(
+            "Random state:",
```

Comment:
We, programmers, know what "Random state" this means, but it may not be understood by muggles. We try to explain these in non-technical terms -- by what it means (replicability...) and not how it's implemented (fixed initial random state).

Reply:
Its is not necessary to enter an initial value as long there is a way to fix it or use the same start value each time. So probalby a checkbox might do the same.

Reply:
I think @janezd was referring to the naming of the checkbox. It should say something like 'Replicable', the way it is done in Data Sampler.

Reply:
I agree. 'Replicable' is fine for me.

BTW: Is there any further action expected from my side, as the  status has been closed?
Shall I open another pullrequest with the code changes or will this be implemented by someone else?

Reply:
> Is there any further action expected from my side

I opened #3724. I suppose it will be implemented on Friday.

However, you can do it if you wish. In this case, consider the comments that I've written as guidelines. Including removing redundant parentheses in conditions. :)

Reply:
Ok, great.
So I will consider your suggestions and do the changes and test it within my application.